### PR TITLE
Adds cucumber testing environment

### PR DESCRIPTION
**Pull request by amandagibson**
- Modify migration files
- Added cucumber and database_cleaner gems
- Ran `rails generate cucumber:install`
- Cleaned up scaffolded files
screenshot 2019-02-12 at 15 33 35

### CODE REVIEW

**Review by NiciP — APPROVED** [review]

**Review by boamatule — APPROVED** [review]

File: app/models/user.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+  :recoverable, :rememberable, :trackable, :validatable
```
Comment (oliverochman) [inline app/models/user.rb:3]: fix indentation here

File: features/support/env.rb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# IMPORTANT: This file is generated by cucumber-rails - edit at your own peril.
+# It is recommended to regenerate this file in the future when you upgrade to a
+# newer version of cucumber-rails. Consider adding your own code to a new file
+# instead of editing this one. Cucumber will automatically load all features/**/*.rb
+# files.
```
Comment (oliverochman) [inline features/support/env.rb]: clean this file up

**Review by yanqiuzhang — APPROVED** [review]

**Review by oliverochman — CHANGES_REQUESTED** [review]
Please add a screenshot of running cucumber in the terminal

**Review by vick3d — APPROVED** [review]

File: app/models/user.rb (hunk 1 above)
Comment (amanda-mae) [reply app/models/user.rb:3]: done

File: features/support/env.rb (hunk 2 above)
Comment (amanda-mae) [reply features/support/env.rb]: done

**Comment by amandagibson** [comment]
screenshot 2019-02-12 at 15 33 35

**Pull request merged**